lib/game.rb

```diff
@@ -1,50 +1,104 @@
-require_relative "console/user_interaction"
-require_relative "language"
-require_relative "board"
-require_relative "tictactoe"
-require_relative "human"
-require_relative "computer"
-
+require_relative 'console/user_interaction'
+require_relative 'language'
+require_relative 'board'
+require_relative 'tictactoe'
+require_relative 'human'
+require_relative 'computer'
+# This class handles the set up of the game
 class Game
-
   def start
+    ask_to_play
+    create_players
+    board = Board.new(@ui, @markers)
+    tictactoe = TicTacToe.new(@players, board, @ui)
+    tictactoe.set_current_player(@hum, @com, @player1, @player2)
+    @current_player = tictactoe.current_player
+    @ui.board
+    board.print_board
+    @ui.first(@current_player.name)
+    tictactoe.play(board)
+  end
+
+  def create_players
+    @markers = Marker.new
+    mode = set_mode
+    @name = set_user_name
+    define_difficulty_level(@name) unless @ui.human(mode)
+    human_vs_computer if @ui.computer_human(mode)
+    human_vs_human if @ui.human(mode)
+    computer_vs_computer if @ui.computer(mode)
+    @players = [@player1, @player2]
+    @hum = @player1.choose_symbol
+    @com = @player2.define_symbol(@hum)
```

Comment:
Given that @player1 and @player2 aren't necessarily human and computer, could we change these to @player1_symbol and @player2_symbol?

Reply:
resolved in ab0ebd9